recipes/zxing-cpp/all/conanfile.py

```diff
@@ -1,11 +1,13 @@
-from conans import ConanFile, tools, CMake
-from conans.errors import ConanInvalidConfiguration
-from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
-import functools
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import is_msvc_static_runtime, is_msvc
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
 import os
 
-required_conan_version = ">=1.43.0"
-
+required_conan_version = ">=1.53.0"
```

Comment:
a few lines below this one is the `homepage` field. It's contents is outdated. It should read `https://github.com/zxing-cpp/zxing-cpp`